### PR TITLE
[FormBundle] Add length validation to string form field

### PR DESCRIPTION
**Pull request by dannyvw**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | comma separated list of tickets fixed by the PR

The max length in the database is 255. Without validation you get an database error.

### CODE REVIEW

**Review by ProfessorKuma — CHANGES_REQUESTED** [review]
Hi @, your PR needs some changes
* It seems that you should have submitted to the latest minor branch.
* This PR seems to need a milestone of a patch release.

File: src/Kunstmaan/FormBundle/Entity/FormSubmissionFieldTypes/StringFormSubmissionField.php [hunk 1]
```diff
@@ -16,6 +17,9 @@ class StringFormSubmissionField extends FormSubmissionField
 {
     /**
      * @ORM\Column(name="sfsf_value", type="string")
+     * @Assert\Length(
+     *     max=255
```
Comment (acrobat) [inline src/Kunstmaan/FormBundle/Entity/FormSubmissionFieldTypes/StringFormSubmissionField.php:21]: minor comment but can you keep it all on one line? Thanks!

**Review by acrobat — CHANGES_REQUESTED** [review]
Minor comment, otherwise good to merge!

**Review by acrobat — APPROVED** [review]

**Review by ProfessorKuma — APPROVED** [review]
Hi @, your PR passed all our requirements.

Thank you for contributing!

**Review by ProfessorKuma — DISMISSED** [review]
Hi @, your PR needs some changes
* It seems that you should have submitted to the latest minor branch.

**Comment by acrobat** [comment]
Thanks @dannyvw!

**Pull request merged**